tango_simlib/utilities/helper_module.py

```diff
@@ -55,6 +58,13 @@
 
 TANGO_NOT_SPECIFIED_PROPS = ['Not specified', 'No display unit', 'No standard unit']
 
+POGO_USER_DEFAULT_CMD_PROP_MAP = {
+        'name': 'name',
+        'arginDescription': 'doc_in',
+        'arginType': 'dtype_in',
+        'argoutDescription': 'doc_out',
+        'argoutType': 'dtype_out'}
```

Comment:
The keys in this dict never get used, so do we need a dict here, or just a tuple?  Also, the old copy in `tango_sim_generator.py` can be dropped now.  There is also one in `sim_xmi_parser.py`, which just has one extra key - `inherited`.  Can share one copy everywhere?